HomeWorks/java/lesson4/StudentTest.java

```diff
@@ -0,0 +1,32 @@
+package lesson4;
+
+public class StudentTest {
+
+
+    public static void main(String[] args) {
+        Student student1 = new Student();
+        student1.mathAverageMark = 2.5;
+        student1.economyAverageMark = 5.5;
+        student1.foreignAverageMark = 1.5;
+        double sum1 = student1.mathAverageMark + student1.economyAverageMark + student1.foreignAverageMark;
```

Comment:
It's better to create a method for an average score calculation because we violate DRY principle which we should follow in programming

Reply:
Solution can be like this:
public void calculateAverageGrade() {
double average = (mathGrade + economicGrade + foreignLanguageGrade) / 3;
System.out.println("Student " + studentFirstName + " " + studentLastName + " has average grade: " + average);
}